src/g_game.c

```diff
@@ -4706,6 +4769,9 @@ void G_BindWeapVariables(void)
   M_BindBool("doom_weapon_toggles", &doom_weapon_toggles, NULL,
              true, ss_weap, wad_no,
              "Allow toggling between SG/SSG and Fist/Chainsaw");
+  M_BindBool("full_weapon_cycle", &full_weapon_cycle, NULL,
```

Comment:
So, this is mostly the same as `doom_weapon_toggles`, but for next/prev instead of pressing 1 or 3?